meta-mender-core/recipes-bsp/grub/grub-efi-mender-precompiled_2.04.bb

```diff
@@ -15,8 +15,8 @@ inherit grub-mender-grubenv
 
 FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 
-LICENSE = "GPL-3.0"
-LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/GPL-3.0;md5=c79ff39f19dfec6d293b95dea7b07891"
+LICENSE = "GPL-3.0-only"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/GPL-3.0-only;md5=c79ff39f19dfec6d293b95dea7b07891"
```

Comment:
I believe this is a mistake in upstream. Grub clearly specifies that you can use a later version as well.